.github/ISSUE_TEMPLATE/cran-release.yml

```diff
@@ -29,67 +29,91 @@ body:
         - [ ] Review old/hanging PRs before going into the release (Optional).
         - [ ] Revisit R-package's lifecycle badges (Optional).
         - [ ] Make sure that all upstream dependencies of this package that need to be submitted to CRAN were accepted before going into release activities.
         - [ ] Make sure integration tests are green 2-3 days before the release. Look carefully through logs (check for warnings and notes).
         - [ ] Decide what gets merged in before starting release activities.
   - type: textarea
-    id: release-checklist
+    id: release
     attributes:
-      label: Release Checklist
+      label: Release
       description: The steps to be taken in order to create a release.
       placeholder: Steps to create a release.
       value: |
+        ### Prepare the release
+        
+        - [ ] Create a new release candidate branch 
+          `git checkout -b release-candidate-vX.Y.Z`
         - [ ] Update NEWS.md file: make sure it reflects a holistic summary of what has changed in the package.
-        - [ ] Remove the additional fields (`Remotes` and `Config/Needs/*`) from the DESCRIPTION file where applicable.
-        - [ ] Increase versioned dependency on {package name} to >=X.X.X (Optional).
+        - [ ] Remove the additional fields (`Remotes`) from the DESCRIPTION file where applicable.
         - [ ] Make sure that the minimum dependency versions are updated in the DESCRIPTION file for the package and its reverse dependencies (Optional).
-        - [ ] Create a pull request to make necessary bug fixes/changes (add "[skip vbump]" in the pr title), and after merging the PR, tag the update(s) as a release candidate v < intended release version > -rc < release candidate iteration > on the main branch.
-        - [ ] Build the package locally using the command:`R CMD build .` which will generate a .tar.gz file necessary for the CRAN submission.
-        - [ ] Submit the package that was build in the previous step via this form: https://cran.r-project.org/submit.html.
-        - [ ] Address CRAN feedback, tag the package vX.X.X-rc(n+1) and repeat the submission to CRAN whenever necessary.
+          - [ ] Increase versioned dependency on {package name} to >=X.Y.Z (Optional).
+        - [ ] Commit your changes and create the PR on GitHub (add "[skip vbump]" in the PR title). Add all updates, commit, and push changes:
+          `# Make the necessary modifications to your files
+          # Stage the changes
+          git add <files your modified>
+          # Commit the changes
+          git commit -m "[skip vbump] <Your commit message>"
+          git push origin release-candidate-vX.Y.Z`
+
+          ### Test the release
+
+        - [ ] Execute the manual tests on Shiny apps that are deployed on various hosting providers (Posit connect and shinyapps.io) - track the results in GitHub issue (Applicable only for frameworks that use Shiny).
```

Comment:
do we have a designated posit cloud account for this? or should the person whoever is doing this to use their own account? for both posit cloud and shinyapps.io